tests/functions/test_memory.py

```diff
@@ -510,7 +510,7 @@ def test_DictionaryMemory_add_and_delete_from_memory(self):
                            [[ 1,  2,  3],[200,201,202,203]]]
         for m,e in zip(em.memory,expected_memory):
             for i,j in zip(m,e):
-                assert np.allclose(i,j)
```

Comment:
Can the zip iterations be replaced by a single call to `np.testing.assert_allclose`?

Reply:
No, also ragged array